cudax/include/cuda/experimental/__memory_resource/any_resource.h

```diff
@@ -95,10 +105,12 @@ class basic_any_resource
     }
   }
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS // Doxygen misparses the below constructor, so dumb it down for it.
+  //! @brief Conversion from a \c basic_any_resource with the same set of properties but in a different order.
```

Comment:
i think we should require properties to have a UUID. that way we can have a canonical global ordering of properties. not for this PR obviously.